assets/css/global.css

```diff
@@ -197,15 +125,15 @@ ul {
 }
 
 strong {
-  @apply font-comme-medium tracking-[0.01em] !text-black-core/[1] !font-medium text-[1.0625rem] md:text-[1.15625rem];
+  @apply font-comme tracking-[0.01em] !text-black-core/[1] !font-medium text-[1.0625rem] md:text-[1.15625rem];
 }
 
 h1 strong {
   @apply text-[1.8rem] lg:text-[2rem];
 }
 
 h2 strong {
-  @apply text-2xl lg:text-[1.7rem] font-comme-semibold;
+  @apply text-2xl lg:text-[1.7rem] font-comme font-semibold;
```

Comment:
is it will be only font-comme-semibold as why this font is needed between

Reply:
Actually, it will use **font-comme-semibold**, but by using next/font, you can see the correct font url in _app.js file, I used the font in such a way by using short name and reducing variable numbers.